### PR TITLE
#102 Add unit tests for buildChecker.py and remove unused script

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

File: dynawo/sources/CMakeLists.txt [hunk 1]
```diff
@@ -38,6 +38,7 @@ if(BUILD_TESTS)
   add_subdirectory(Modeler/DataInterface/test)
   add_subdirectory(Models/CPP/ModelOmegaRef/test)
   add_subdirectory(Models/CPP/ModelNetwork/test)
+  ADD_SUBDIRECTORY(Models/Modelica/PreassembledModels/test)
```
Comment (gautierbureau) [inline dynawo/sources/CMakeLists.txt]: small case

File: dynawo/sources/Models/Modelica/PreassembledModels/test/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+SET(MODULE_NAME MODELS_PREASSEMBLED_unittest)
```
Comment (gautierbureau) [inline dynawo/sources/Models/Modelica/PreassembledModels/test/CMakeLists.txt]: small case in all the file for cmake commands

File: dynawo/sources/CMakeLists.txt (hunk 1 above)
Comment (rosiereflo) [reply dynawo/sources/CMakeLists.txt]: Solved

File: dynawo/sources/Models/Modelica/PreassembledModels/test/CMakeLists.txt (hunk 2 above)
Comment (rosiereflo) [reply dynawo/sources/Models/Modelica/PreassembledModels/test/CMakeLists.txt]: Solved

**Pull request merged**